src/components/TopicList/index.js

```diff
@@ -1,24 +1,31 @@
 import React from 'react';
 import { FlatList, View } from 'react-native';
-import { array } from 'prop-types';
+import { array, func } from 'prop-types';
 
 import ListItem from 'components/TopicListItem';
 import styles from './styles';
 
-const TopicList = ({ list }) => {
+const TopicList = ({ list, onPress }) => {
```

Comment:
```suggestion
const TopicList = ({ list, onPress }) => (
```
and remove the `return` on the next line.